### PR TITLE
Bug 478040 - [api] We need a way to obtain the current versions we use

### PR DESCRIPTION
**Pull request by dhuebner**
to generate build files

Signed-off-by: Dennis Huebner <dennis.huebner@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=478040

**Comment by dhuebner** [comment]
Jenkins intellij-tests-master-latest

File: plugins/org.eclipse.xtext.util/src/org/eclipse/xtext/util/XtextVersion.xtend [hunk 1]
```diff
@@ -6,10 +6,15 @@ import java.net.URL
...
 import org.eclipse.emf.common.EMFPlugin
 import org.eclipse.xtend.lib.annotations.Data
-
+/**
+ * @since 2.9
+ */
 @Data
 class XtextVersion {
 	
+	public static String XTEND_GRADLE_PLUGIN_VERSION  = '0.4.8'
```
Comment (oehme) [inline plugins/org.eclipse.xtext.util/src/org/eclipse/xtext/util/XtextVersion.xtend]: We should make these dependent on the Xtext version, not constants.

Comment (dhuebner) [inline plugins/org.eclipse.xtext.util/src/org/eclipse/xtext/util/XtextVersion.xtend]: Feel free to change it

**Pull request merged**